ARMA3.cs/ARMA3.cs

```diff
@@ -28,13 +28,13 @@ public class ARMA3 : SteamCMDAgent // SteamCMDAgent is used because ARMA3 relies
 
 
         // - Settings properties for SteamCMD installer
-        public override bool loginAnonymous => false; // ARMA3 requires to login steam account to install the server, so loginAnonymous = false
+        public override bool loginAnonymous => true; // ARMA3 DOES NOT require to login steam account to install the server, so loginAnonymous = true
         public override string AppId => "233780"; // Game server appId, ARMA3 is 233780
 
 
         // - Game server Fixed variables
-        public override string StartPath => "arma3server.exe"; // Game server start path, for ARMA3, it is arma3server.exe
-        public string FullName = "Arma 3 Dedicated Server"; // Game server FullName
+        public override string StartPath => "arma3server_x64.exe"; // Game server start path, for ARMA3, it is arma3server_x64.exe
```

Comment:
This assumes that the computer hosting the server is 64 bit no? 

Can WindowsGSM be run on 32-bit (x86) OS's? Or would a WindowsGSM user realistically want to run 32-bit server?

Reply:
Probably could be run on 32-bit but realistically speaking the Arma 3 process itself uses more than 4 GB of RAM. 
32 bit Windows can only use a maximum of 4 GB RAM,  windows server itself uses at a minimum 1.2GB RAM (non server version even more), leaving less than 2.8GB RAM for Arma.

So highly unlikely I'd say and definitely not the majority use case.